build-aux/snap/snapcraft.yaml

```diff
@@ -323,6 +323,10 @@ parts:
       # systemd package will those, they have to continue to match our
       # paths. We force the value here in case we change the default.
       make -C data install DESTDIR="${CRAFT_PART_INSTALL}" SYSTEMDUSERUNITDIR=/usr/lib/systemd/user SYSTEMDSYSTEMUNITDIR=/lib/systemd/system
+
+      # We do not pack snap-bootstrap. We should not pack its service files.
+      rm "${CRAFT_PART_INSTALL}/lib/systemd/system/snapd.recovery-chooser-trigger.service"
```

Comment:
it may be the case that it's not shipped by the base at all and it only appears in the system when snapd snap is seeded.